bin/polly.py

```diff
@@ -249,12 +249,12 @@ def PositiveInt(string):
   build_tag = polly_toolchain
 
 """Tune environment"""
-if toolchain_entry.name == 'mingw' or toolchain_entry.name == 'mingw-c11':
+if toolchain_entry.name == 'mingw' or toolchain_entry.name == 'mingw-c11' or toolchain_entry.name == 'mingw-cxx14' or toolchain_entry.name == 'mingw-cxx17':
```

Comment:
Since there are a lot of toolchains here it make sense to use `startswith('mingw')`.